client/room.html

```diff
@@ -206,27 +206,36 @@ <h1>Share link</h1>
       <button id="shareOK">OK</button>
     </div>
 
-    <div id="aboutOverlay" class="overlay">
-      <h1>About</h1>
+    <div id="aboutOverlay" class="overlay" style = overflow:scroll>
```

Comment:
Put the style into the stylesheet, please.

Reply:
done

Reply:
The style sheet thing was my fault.  I don't know how all this works.  I just know that when I added lots of text to the overlay, it needed a scrollbar and I figured out how to add one.  Thanks for fixing, Raphael.

Reply:
Yeah no problem. Don't worry nobody was pointing fingers